contracts/deposit-service/ReceiverImplementation.sol

```diff
@@ -61,16 +63,17 @@ contract ReceiverImplementation is DepositServiceBase {
             return;
         }
 
-        address wrappedTokenAddress = wrappedToken();
         uint256 amount = address(this).balance;
 
         if (wrappedTokenAddress == address(0)) revert WrappedTokenNotSupported();
+        // slither-disable-next-line incorrect-equality
```

Comment:
why this warning?

Reply:
I think it's a false positive because we are comparing to 0 in this case. Please read more about this check here
https://github.com/crytic/slither/wiki/Detector-Documentation#dangerous-strict-equalities